third_party/proton/proton/specs.py

```diff
@@ -11,11 +11,25 @@
         lambda width, num_sms, clock_rate, **kwargs: (num_sms * 16384 * (clock_rate / 1e3) * 1e6) / (width / 8),
     },
     "HIP": {
-        "gfx90a": lambda width, **kwargs: 383e12 / (width / 8),
-        "gfx942": lambda width, **kwargs: 2614.9e12 / (width / 8),
+        "gfx90a":
+        lambda width, **kwargs: 383e12 / (width / 8),
+        "gfx942":
+        lambda width, **kwargs: 2614.9e12 / (width / 8),
+        "gfx950":
+        lambda width, num_sms, clock_rate, model_name, **kwargs:
+        (amd_fp8_matrix_flops_per_cu[model_name] * num_sms * clock_rate * 1e3) / (width / 8),
     },
 }
 
+amd_bps_by_arch = {
+    'gfx90a': 3.2 * 1e12,
+    'gfx942': 5.3 * 1e12,
+    'gfx950': 8.0 * 1e12,
+}
+
+# FP8 Matrix FP8 Performance(FLOPS/clock/CU)
+amd_fp8_matrix_flops_per_cu = {'MI350X': 8192, 'MI355X': 8192}
```

Comment:
I think we don't need to use model name here--the flops per clock per cu is a CU characteristic; which is captured entirely with `gfx` target. So `amd_fp8_flops_by_arch` and use `gfx` target as key is good.